packages-exp/auth-exp/src/core/user/token_manager.ts

```diff
@@ -71,6 +89,26 @@ export class StsTokenManager {
     };
   }
 
+  private async refresh(auth: Auth, oldToken: string): Promise<void> {
+    const { accessToken, refreshToken, expiresIn } = await requestStsToken(
+      auth,
+      oldToken
+    );
+    this.updateTokensAndExpiration(accessToken, refreshToken, expiresIn);
+  }
+
+  private updateTokensAndExpiration(
+    accessToken: string | undefined,
+    refreshToken: string | undefined,
+    expiresInSec: string | undefined
```

Comment:
can we push this | undefined up a level?

Reply:
Done, still needs to be `string | null` though